### PR TITLE
Initial revision of SLSA blog post.

### PR DESCRIPTION
**Pull request by bendory**
# Changes

Added SLSA blog post.

This blog initially appeared at https://opensource.googleblog.com/2023/03/getting-to-slsa-level-2-with-tekton-and-tekton-chains.html

Note: do not merge this PR until the corresponding example has been merged: https://github.com/tektoncd/chains/pull/775

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [N/A] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

### CODE REVIEW

**Comment by bendory** [comment]
Preview is now available at: https://deploy-preview-537--tekton.netlify.app/blog/2023/04/17/getting-to-slsa-level-2-with-tekton-and-tekton-chains/

**Comment by bendory** [comment]
/do-not-merge

confirm that this PR merges first: https://github.com/tektoncd/chains/pull/775

**Review by geriom — APPROVED** [review]

**Review by geriom — CHANGES_REQUESTED** [review]
Meant to request changes instead of approve 😅

File: content/en/blog/2023/slsa-2/index.md [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+   “build script”. The only manual command, if any, is to invoke the build
+   script.
+1. [Provenance][available]: The provenance is available to the consumer in a
+   format that the consumer accepts. The format SHOULD be in-toto SLSA
+   Provenance, but another format MAY be used if both producer and consumer
+   agree and it meets all the other requirements.
+
+Tekton Tasks, TaskRuns, Pipelines and PipelineRuns are specified in yaml files. These yaml files can be considered as scripts and can even be checked in into a code repository. These could also be run from code repositories. Tekton Chains provides a way to generate provenance in in-toto SLSA format. As such, Tekton can easily make builds which satisfy the SLSA L1 requirements.
+
+Let's follow through with [an example][example-repo], which has the following files:
```
[the remark this answers is not in the document]
Comment (bendory) [reply content/en/blog/2023/slsa-2/index.md:58]: 👍

**Review by geriom — APPROVED** [review]
Thank you!

**Pull request merged**